nextjs-app-v14/src/app/pages/cookie-banner/page.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+
+import NavigationLayout from "@/layout/navigation";
+import { PieCookieBanner } from "@justeattakeaway/pie-cookie-banner/dist/react";
+// TODO: Remove this comment as soon as we provide the TS declaration for locales
+// @ts-ignore: missing declaration for locales
+import locales from "@justeattakeaway/pie-cookie-banner/locales"
+
+export default function CookieBanner() {
+    return (
+        <NavigationLayout title="Cookie Banner">
+        <PieCookieBanner
+            defaultPreferences={{functional: true, personalized: true, analytical: true}}
+            locale={locales.enGB}
```

Comment:
I think for the other apps we updated to use the Danish locale so that we can be sure it's not just using the default